index.html

```diff
@@ -13,19 +13,11 @@
       data-sdk-integration-source="integrationbuilder_ac"
     ></script>
     <script>
-      const FUNDING_SOURCES = [
-        paypal.FUNDING.PAYPAL,
-        paypal.FUNDING.PAYLATER,
-        paypal.FUNDING.VENMO,
-        paypal.FUNDING.CARD,
-      ];
-
-      FUNDING_SOURCES.forEach((fundingSource) => {
         const button = paypal.Buttons({
-          fundingSource,
           style: {
             shape: 'rect',
-            color: fundingSource === paypal.FUNDING.PAYLATER ? 'gold' : '',
+            //color:'blue' change the default color of the buttons
```

Comment:
This is super minor, but can we have a space after the comment to match the other comments in this file?
```suggestion
            // color:'blue' change the default color of the buttons
```